app/assets/stylesheets/darkswarm/_shop-navigation.css.scss

```diff
@@ -60,17 +60,25 @@ ordercycle {
     }
 
     select {
+      background-image: url('/assets/white-caret.svg');
```

Comment:
would you mind exploring how to leverage Rails' asset hashing in https://guides.rubyonrails.org/asset_pipeline.html#css-and-sass ? That's the latest version of the guides and haven't found that same info in v3.2 but they claim we can use ERB as usual. Can you give it a try? our servers will appreciate it.

Reply:
nice one, I prefer to move this PR forward immediately and get it tested asap, I created an issue for your point:
https://github.com/openfoodfoundation/openfoodnetwork/issues/5378
If it works we can replace the other cases of url('/assets/